services/catarse/catarse.js/legacy/src/root/projects-subscription-thank-you.tsx

```diff
@@ -95,69 +95,67 @@ const ProjectsSubscriptionThankYou = {
         return m('#thank-you', !project ? h.loader() : [
             m('.page-header.u-marginbottom-30',
                 m('.w-container',
-                    m('.w-row',
-                        m('.w-col.w-col-10.w-col-push-1', [
-                            m('.u-marginbottom-20.u-text-center',
-                                projectUser ? m(`img.big.thumb.u-round[src='${projectUser.profile_img_thumbnail}']`) : h.loader()
+                    m('.thanks-header',
+                        m('.thanks-header-title-wrapper', [
+                            m('.thanks-thumb-wrapper',
+                                projectUser ? m(`img.thumb.u-round[src='${projectUser.profile_img_thumbnail}']`) : h.loader()
```

Comment:
Alteração da tela.

Reply:
@gabrielras Eu validei em Sandbox e aqui ainda continua com um erro. Pelo que eu vi aqui, pode ter a ver com você ter removido o state.isEdit, o que faz com que o Thank You de assinaturas esteja sendo sempre o mesmo, independente de ser uma edição de assinatura ou um início de assinatura. Veja aqui que eu explico melhor:
https://www.loom.com/share/1e242af4cf674b58994e4d080bb36f07

Qualquer coisa, veja com o @grpse que ele pode te ajudar nesse ponto aqui